cmd/internal/policy/os.go

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
This was added to mock `os.MkdirTemp`, but the test for that is no longer needed (and was in an earlier revision of this that you could not have seen) so it can be removed.